application/backend/src/main/resources/apps/typerefinery/components/widgets/chart/clientlibs/variants/barchart/functions.js

```diff
@@ -0,0 +1,229 @@
+window.Typerefinery = window.Typerefinery || {};
+window.Typerefinery.Components = Typerefinery.Components || {};
+window.Typerefinery.Theme = Typerefinery.Theme || {};
+window.Typerefinery.Components.Widgets = Typerefinery.Components.Widgets || {};
+window.Typerefinery.Components.Widgets.Charts = Typerefinery.Components.Widgets.Charts || {};
+window.Typerefinery.Components.Widgets.Charts.Variants = Typerefinery.Components.Widgets.Charts.Variants || {};
+window.Typerefinery.Components.Widgets.Charts.Variants.BarChart = Typerefinery.Components.Widgets.Charts.Variants.BarChart || {};
+window.Typerefinery.Components.Widgets.Charts.Instances = Typerefinery.Components.Widgets.Charts.Instances || {};
+
+(function (ns, typerefineryNs, componentNs, themeNs, chartInstanceNs, document, window) {
+    "use strict";
+
+    const DEFAULT_DATA = {
```

Comment:
should this be added to namespace?

Reply:
If I add this out side the name space and pass as an arg we can use the same name in all the functions.js

Got "DEFAULT_DATA Already used"